src/library/trackset/baseplaylistfeature.cpp

```diff
@@ -16,11 +16,17 @@
 #include "library/trackcollection.h"
 #include "library/trackcollectionmanager.h"
 #include "library/treeitem.h"
+#include "library/treeitemmodel.h"
 #include "track/track.h"
 #include "util/assert.h"
 #include "widget/wlibrary.h"
+#include "widget/wlibrarysidebar.h"
 #include "widget/wlibrarytextbrowser.h"
 
+namespace {
+auto kUnsafeFilenameReplacement = QLatin1String("-");
```

Comment:
```suggestion
constexpr QChar kUnsafeFilenameReplacement = '-';
```
What was the issue with this proposal? 
constexpr ensures that this is a constant in the text segment. 
QChar is already the 16 bit representation of the char, this avoids conversion at runtime.
replace() needs the 16 bit representation. 

It is different story when comparing strings. In this case Qt performs a 8 bit char compare which is faster than a 16 bit compare and a Latin1String() is the fates alternative then.

Reply:
I see, fixed